knime_extension/src/nodes/conversion.py

```diff
@@ -0,0 +1,755 @@
+            knext.Port(
+                type=knext.PortType.TABLE,
+                name="Geo table",
+                description="Input table with additional geometry column in the units of the provided CRS.",
+            )
+        ]
+
+    def configure(self, configure_context, input_schema):
+        # input_column and crs need to be defined in the child class
+        self.input_column = knut.column_exists_or_preset(
+            configure_context, self.input_column, input_schema, self.column_type_check
+        )
+        validate_crs(self.crs)
+
+        import knime.types.geospatial as gt
+
+        return input_schema.append(
+            knext.Column(
+                ktype=knext.logical(gt.GeoValue),
+                name=knut.get_unique_column_name(self.DEF_GEO_COL_NAME, input_schema),
+            )
+        )
+
+    def execute(self, exec_context, input_table):
+        # create GeoDataFrame with the selected column as active geometry column
+        df = input_table.to_pandas()
+        try:
+            # input_column and crs need to be defined in the child class
+            result_col_name = knut.get_unique_column_name(
+                self.DEF_GEO_COL_NAME, input_table.schema
+            )
+            df[result_col_name] = self.converter(df, self.input_column)
+            gdf = gp.GeoDataFrame(df, geometry=result_col_name, crs=self.crs)
+        except Exception as error:
+            raise RuntimeError(
+                f"Error converting '{self.input_column}' column to geometry: {str(error)}"
+            )
+        # execute the function and append the result
+        return knut.to_table(gdf, exec_context)
+
+
+############################################
+# WKT to Geometry
+############################################
+@knext.node(
+    name="WKT to Geometry",
+    node_type=knext.NodeType.MANIPULATOR,
+    icon_path=__NODE_ICON_PATH + "WKTtoGeo.png",
+    category=__category,
+    after="",
+)
+@knut.geo_node_description(
+    short_description="Converts the input Well-known-text (WKT) column to a geometry column.",
+    description="""This node converts the selected 
+    [Well-known-text (WKT)](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) input column to 
+    a geometry column in the units of the provided CRS.
+    """,
+    references={
+        "From WKT": "https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.from_wkt.html",
+        "CRS parser from pyproj": "https://pyproj4.github.io/pyproj/stable/api/crs/crs.html#pyproj.crs.CRS.from_user_input",
+    },
+)
+class WKTtoGeoNode(_ToGeoConverter):
+
+    input_column = knext.ColumnParameter(
+        label="WKT column",
+        description="[Well-known-text (WKT)](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) column to convert",
+        column_filter=knut.is_string,
+        include_row_key=False,
+        include_none_column=False,
+    )
+
+    crs = crs_input_parameter()
```

Comment:
Ah I see. Here it would be nice if we provided a way to derive the parameters from the parent class. If there were more than a single parameter, this could be solved via composition instead of inheritance using a parameter group. But for the case you have here, we don't offer a simpler solution yet.

Reply:
Wait... When thinking about it a little more I actually don't see a reason why it wouldn't work to put `crs` into `_ToGeoConverter`. To find all those parameter "descriptors" (the Python lingo for the construct we're using) we simply iterate over all attributes defined on the class, and that should be the combined set of parent and child class. Have you tried moving crs one level up?

Reply:
Tried it and the parameter no longer shows up in the node dialog. Will leave it like it is for now.